### ISSUE TITLE
How to detect fluorescence

### ISSUE THREAD

**Issue by strikecity** [strikecity/RTSpec#3]
RE: UN-001
This will be difficult to do since the photodiode will detect both the 485 nm excitation as well as the 520 nm fluorescent emission.

**Comment by strikecity** [comment]
Can we just put a piece of translucent green plastic over the photodiode (sensor)?

**Comment by TheVetter** [comment]
Are you talking about a band pass  filter to allow only a range of wavelengths through?

**Comment by strikecity** [comment]
Yes. Meredith wants to use GFP, which stands for Green Fluorescent Protein. It absorbs a non-visible wavelength and emits Green.

**Comment by TheVetter** [comment]
we would then need two photodiodes for the device. one for Raquibs part and one for the GFP.

**Comment by strikecity** [comment]
Good point. Yes we would. But, this is better than a detector which cannot differentiate between the non-visible excitation wavelength and well as the green fluorescence. Alternatively, we could think about a shutter mechanism to extend/retract the green filter, but that might be overly complicated and failure prone.

**Comment by strikecity** [comment]
Dr. Ben Brooks has shared a link to this company which makes optical filters: http://www.knightoptical.com/

**Comment by TheVetter** [comment]
Possibly our best option for the filters. http://www.knightoptical.com/stock/optical-components/uvvisnir-optics/filters/dichroic-filters/dichroic-longpass-filters/dichroic-filter-longpass-510nm-dia-125mm/

it will allow a small amount of the 485 through  but I think it will work. Also the Shipping cost is $20 for 2 week delivery or $40 for 2 day shipping. The cost is so high because there warehouse is in the UK.

**Comment by strikecity** [comment]
Discussed this w/ Drs. Brooks on 11/21 and agreed to order the filter recommended by Andrew (above). Andrew will process the order.

**Comment by TheVetter** [comment]
We now have a way to detect 520nm with our current photodiode but we now need 485nm light source to excite the bacteria.
